frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/MembersPage/MembersPage.tsx

```diff
@@ -286,6 +286,13 @@ const MembersPage = ({ location, router }: IMembersPageProps): JSX.Element => {
             setCreateUserErrors({
               email: "A user with this email address has already been invited",
             });
+          } else if (
+            userErrors.data.errors?.[0].reason.includes("password too long")
+          ) {
+            renderFlash(
+              "error",
+              "Could not create user. Password is over the 48 characters limit. If the password is under 48 characters, please check the auth_salt_key_size in your Fleet server config."
```

Comment:
Should it not be 72?

Reply:
72 is the max length of the argument passed to the hashing function. We add a salt to the plaintext password, so plaintext + salt must be < 72. Default salt length is 24, so default max plaintext length is 48.

Reply:
Got it!